packages/fetch/src/index.js

```diff
@@ -0,0 +1,69 @@
+/**
+ * Internal dependencies
+ */
+import createNonceMiddleware from './middlewares/nonce';
+import createRootURLMiddleware from './middlewares/root-url';
+import createPreloadingMiddleware from './middlewares/preloading';
+import namespaceEndpointMiddleware from './middlewares/namespace-endpoint';
+import httpV1Middleware from './middlewares/http-v1';
+
+const middlewares = [];
+
+function registerMiddleware( middleware ) {
+	middlewares.push( middleware );
+}
+
+function request( options ) {
```

Comment:
Why is it called `request`? Does this map to `fetch` somewhere?

Reply:
It's just an internal name to avoid confusion with `window.fetch` (which can also be accessed by `fetch` since it's a global) used on the same file.

Reply:
Maybe it should have been called `apifetch` then. Or `wpfetch`.